targets/stm32l0x2.json

```diff
@@ -0,0 +1,22 @@
+{
+    "inherits": [
+        "cortex-m"
+    ],
+    "llvm-target": "armv6m-none-eabi",
+    "build-tags": [
+        "stm32l0",
+        "stm32l0x2",
+        "stm32"
+    ],
+    "cflags": [
+        "--target=armv6m-none-eabi",
+        "-Qunused-arguments"
+    ],
+    "linkerscript": "targets/stm32l072czt6.ld",
```

Comment:
Because this linker script is for a more specific chip (with a given amount of memory), it shouldn't be included in the more generic chip family specification.